presto-product-tests/src/main/resources/sql-tests/testcases/color_functions/bar.sql

```diff
@@ -1,2 +0,0 @@
--- database: presto; groups: color;
-SELECT bar(1, 10), bar(1, 10, rgb(255, 0, 0), color('#0f0'))
```

Comment:
Is there any sql test for color functions? I see only unit tests `TestColorFunctions`. Maybe those unit tests should be rewritten to actually execute some sql.